doc/benchmarks.rst

```diff
@@ -57,6 +57,24 @@ depending on the Python version.
    them, and more generally to not modify them.
 
 
+async_tree
+----------
+
+Async workload benchmark, which calls asyncio.gather() on a tree (6 levels deep, 6
+branches per level) with the leaf nodes simulating some [potentially] async work
+(depending on the benchmark variant). Available variants:
+
+* ``async_tree``: (also the ``async_tree_no_suspension`` variant) no actual async
+  work at any leaf node.
+* ``async_tree_suspense_all``: all leaf nodes simulate async IO workload (async 
+  sleep 50ms).
```

Comment:
This may be a frivolous request, but can we avoid using the term "suspense" or "suspension"? Especially "suspense" means something different than "suspend" and it makes me think of something like a suspenseful movie.